umh.docs.umh.app/content/en/docs/datacontracts/_index.md

```diff
@@ -0,0 +1,213 @@
+---
+title: "Data Contracts / API"
+menuTitle: "Data Contracts / API"
+description: "This page describes how messages flow in the UMH, which message goes where, how it has to be formatted and how you can create your own structures."
+weight: 1725
+---
+
+## What are Data Contracts
+
+Data Contracts are formal agreements that define the structure, format, and
```

Comment:
this sounds a little bit generic and too chatgpt:
- "By establishing clear
guidelines for data exchange, Data Contracts facilitate seamless integration
and prevent disruptions caused by changes in data formats or structures." --> almost meaningless